testonly/integration/logenv.go

```diff
@@ -83,6 +84,11 @@ type LogEnv struct {
 // TODO(codingllama): Remove 3rd parameter (need to coordinate with
 // github.com/google/certificate-transparency-go)
 func NewLogEnv(ctx context.Context, numSequencers int, _ string) (*LogEnv, error) {
+	return NewLogEnvWithGrpcOptions(ctx, numSequencers, nil, nil)
+}
+
+// NewLogEnvWithGrpcOptions works the same way as NewLogEnv, but allows callers to also set additional grpc.ServerOption and grpc.DialOption values.
+func NewLogEnvWithGrpcOptions(ctx context.Context, numSequencers int, serverOpts []grpc.ServerOption, clientOpts []grpc.DialOption) (*LogEnv, error) {
```

Comment:
NewLogEnvWithGRPCOptions (https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
Thanks for sharing this. I didn't know that.

Done.